src/flag_gems/ops/all.py

```diff
@@ -21,7 +21,11 @@ def reduce_all(a, b):
 
 
 @libentry()
-@triton.autotune(configs=runtime.get_tuned_config("all"), key=["M", "N"])
+@libtuner(
+    configs=runtime.get_tuned_config("naive_reduction"),
+    key=["M", "N"],
+    share="naive_reduction",
```

Comment:
is there any case where the key in `get_tuned_config` is  "native_reduction", but the corresponding key in the DB is something else?
If not, then perhaps we don’t need to add the "share" parameter?